lib/core/src/Network/Ntp.hs

```diff
@@ -118,16 +118,16 @@ instance DefineSeverity NtpTrace where
         NtpTraceIPv4IPv6NoReplies -> Info
         NtpTraceReportPolicyQueryFailed -> Alert
         NtpTraceQueryResult _ -> Info
-        NtpTraceRunProtocolError _ _ -> Alert
+        NtpTraceRunProtocolError _ _ -> Info
         NtpTraceRunProtocolNoResult _ -> Info
         NtpTraceRunProtocolSuccess _ -> Debug
         NtpTraceSocketOpen _ -> Debug
         NtpTraceSocketClosed _ -> Debug
         NtpTracePacketSent _ -> Debug
-        NtpTracePacketSentError _ _ -> Alert
-        NtpTracePacketDecodeError _ _-> Alert
+        NtpTracePacketSentError _ _ -> Debug
+        NtpTracePacketDecodeError _ _-> Debug
         NtpTracePacketReceived _ -> Debug
-        NtpTraceWaitingForRepliesTimeout _ -> Alert
+        NtpTraceWaitingForRepliesTimeout _ -> Info
```

Comment:
It feels a bit weird to have errors reported as Debug and Info. The timeout should at least be a `Notice` IMO, and the Error at least `Error`? What's the rationale behind this change?

Reply:
Currently there are some errors reported (from the underneath lib) on successful request -> https://github.com/input-output-hk/cardano-wallet/issues/1394#issuecomment-594496618. :man_shrugging:

Reply:
These two "Alert" could / should be turned to `Notice` or `Warning` IMO. `Info` or `Debug` is a big extreme.

Reply:
Agree.

The code tries multiple servers and fails if there are less that 3 results returned.

So log messages relating to an individual server failing to query (e.g. ipv6 misconfiguration) could be warning (or notice if you're feeling generous). It's not completely failed yet at this point.

If the whole ntpQuery fails due to too few results, it will trace NtpTraceNoLocalAddr, NtpTraceIPv4IPv6NoReplies or NtpTraceReportPolicyQueryFailed. These are proper Errors, which prevent the API endpoint from returning a result.

Reply:
On my machine, these 2 gave me Alerts in logs on successful request:

NtpTraceRunProtocolError
NtpTracePacketSentError

```
[cardano-wallet.ntp-client:Alert:46] [2020-03-06 09:49:44.90 UTC] ntp client experienced error Network.Socket.ByteString.sendManyTo: does not exist (Network is unreachable) when sending packet using IPv6
[cardano-wallet.ntp-client:Alert:40] [2020-03-06 09:49:44.90 UTC] ntp client experienced error Network.Socket.ByteString.sendManyTo: does not exist (Network is unreachable) when using IPv6 protocol
```

Turned them into Warnings. @rvl suggested, that the errors could be genuine -> https://github.com/input-output-hk/cardano-wallet/issues/1394#issuecomment-595557011), so disabled ipv6 and, indeed, warnings disappeared.

In the end this is how it looks like:

```
		NtpTraceRunProtocolSuccess _ -> Debug
		NtpTraceSocketOpen _ -> Debug
		NtpTraceSocketClosed _ -> Debug
		NtpTracePacketSent _ -> Debug
		NtpTracePacketReceived _ -> Debug
		NtpTraceStartNtpClient -> Info
		NtpTraceTriggerUpdate -> Info
		NtpTraceRestartDelay _ -> Info
		NtpTraceRestartingClient -> Info
		NtpTraceClientSleeping -> Info
		NtpTraceClientStartQuery -> Info
		NtpTraceIPv4IPv6NoReplies -> Info
		NtpTraceQueryResult _ -> Info
		NtpTraceRunProtocolNoResult _ -> Info
		NtpTraceRunProtocolError _ _ -> Warning
		NtpTracePacketSentError _ _ -> Warning
		NtpTracePacketDecodeError _ _-> Warning
		NtpTraceWaitingForRepliesTimeout _ -> Warning
		NtpTraceIOError _ -> Alert
		NtpTraceLookupServerFailed _ -> Alert
		NtpTraceNoLocalAddr -> Alert
		NtpTraceReportPolicyQueryFailed -> Alert
```
Pls suggest if there something that should be changed.

Reply:
So, on second thought looking at all these cases with more care, all the `Alert` ought to to `Warning` or `Notice`? I mean, `Alert` pretty much means: wake people up in the middle of the night to go look at what's going on with the application. 

Here, it's really just the NTP check that didn't got a response from the central NTP server. Nothing's really alarming with that, the wallet can run just fine :)

Reply:
agree that `Notice` is the best fit here as it is the closest to the definition of Notice  `needs attention; something not progressing properly`

Reply:
`Alert` is by definition much more severe : `a subsystem is no longer operating correctly, likely requires manual intervention`

Reply:
Allrighty, turned to `Notice`:

```
        NtpTraceRunProtocolSuccess _ -> Debug
        NtpTraceSocketOpen _ -> Debug
        NtpTraceSocketClosed _ -> Debug
        NtpTracePacketSent _ -> Debug
        NtpTracePacketReceived _ -> Debug
        NtpTraceStartNtpClient -> Info
        NtpTraceTriggerUpdate -> Info
        NtpTraceRestartDelay _ -> Info
        NtpTraceRestartingClient -> Info
        NtpTraceClientSleeping -> Info
        NtpTraceClientStartQuery -> Info
        NtpTraceIPv4IPv6NoReplies -> Info
        NtpTraceQueryResult _ -> Info
        NtpTraceRunProtocolNoResult _ -> Info
        NtpTraceRunProtocolError _ _ -> Notice
        NtpTracePacketSentError _ _ -> Notice
        NtpTracePacketDecodeError _ _-> Notice
        NtpTraceWaitingForRepliesTimeout _ -> Notice
        NtpTraceIOError _ -> Notice
        NtpTraceLookupServerFailed _ -> Notice
        NtpTraceNoLocalAddr -> Notice
        NtpTraceReportPolicyQueryFailed -> Notice
```